src/utils/connect.ts

```diff
@@ -140,7 +140,13 @@ export const encodeBase64UrlMacaroon = (macaroonHex: string): string => {
 
 export const getProtoVersion = async (providerEndpoint: string): Promise<string> => {
   try {
-    const res = await fetch(`https://${providerEndpoint}/v1/info`, {
+    let infoRequestUrl: string;
+    if ((window as any).IS_PACKAGED) {
+      infoRequestUrl = `${window.location.protocol}//${window.location.host}/v1/info`;
+    } else {
+      infoRequestUrl = `https://${providerEndpoint}/v1/info`;
```

Comment:
are we sure it always defaults to https? ie. if the connect URL has no_tls and no_macaroons